### PR TITLE
Feat(Query): ROS Stack Without Template for Alicloud Terraform

### PR DESCRIPTION
**Pull request by cxAndreFelicidade**
**Proposed Changes**
- Query: ROS Stack Without Template for Alicloud Terraform

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/alicloud/ros_stack_without_template/query.rego [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package Cx
+
+CxPolicy[result] {
+	resource := input.document[i].resource.alicloud_ros_stack[name]
+
+	not resource.template_body
+	not resource.template_url
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/ros_stack_without_template/query.rego:7]:
```suggestion
    not common_lib.valid_key(resource, "template_body")
    not common_lib.valid_key(resource, "template_url")
```

Do not forget to import the common lib, please: `import data.generic.common as common_lib`

Please, also update in `kics\assets\queries\terraform\aws\stack_without_template\query.rego`

File: assets/queries/terraform/alicloud/ros_stack_without_template/query.rego [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_ros_stack[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "Attribute 'template_body' or Attribute 'template_url' is set",
+		"keyActualValue": "Both Attribute 'template_body' and Attribute 'template_url' are undefined",
+	}
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/ros_stack_without_template/query.rego:15]: Add `searchLine`, please

File: assets/queries/terraform/alicloud/ros_stack_without_template/metadata.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "id": "92d65c51-5d82-4507-a2a1-d252e9706855",
+  "queryName": "Ros Stack Without Template",
```
Comment (rafaela-soares) [inline assets/queries/terraform/alicloud/ros_stack_without_template/metadata.json:3]:
```suggestion
  "queryName": "ROS Stack Without Template",
```

(for standard propuses)

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/alicloud/ros_stack_without_template/query.rego [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+	resource := input.document[i].resource.alicloud_ros_stack[name]
+
+	not resource.template_body
+	not resource.template_url
+
+	result := {
+		"documentId": input.document[i].id,
+		"searchKey": sprintf("alicloud_ros_stack[%s]", [name]),
+		"issueType": "MissingAttribute",
+		"keyExpectedValue": "Attribute 'template_body' or Attribute 'template_url' is set",
```
Comment (joaorufi) [inline assets/queries/terraform/alicloud/ros_stack_without_template/query.rego:13]:
```suggestion
		"keyExpectedValue": "Attribute 'template_body' or Attribute 'template_url' to be set",
```

**Review by joaorufi — CHANGES_REQUESTED** [review]

**Review by rafaela-soares — APPROVED** [review]
LGMT 🚀

**Pull request merged**